core/src/main/java/com/airbnb/aerosolve/core/transforms/DefaultStringTokenizerTransform.java

```diff
@@ -15,8 +15,6 @@
  * "generateBigrams" specifies whether bigrams should also be generated
  */
 public class DefaultStringTokenizerTransform implements Transform {
```

Comment:
I implemented the DefaultStringTokenizerTransform using NgramTransform since it is essentially a 1gram transform with an optional 2gram transform in a separate output. Changing name to bigram doesn't really make sense since that is just an optional functionality of it (main functionality is just to tokenize strings) and should be deprecated in the future since NgramTransform exists now.